frontend/src/metabase/components/DateRelativeWidget/DateRelativeWidget.tsx

```diff
@@ -106,10 +106,12 @@ export class PredefinedRelativeDatePicker extends Component<PredefinedRelativeDa
                   ButtonsS.Button,
                   ButtonsS.ButtonNormal,
                   ButtonsS.ButtonMedium,
-                  CS.textNormal,
```

Comment:
removed since they does nothing, and to make the style consistent with classes from lines 149 (new)